TUT_BY_page.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace AutomationTut.by
+{
+    public class TUT_BY_page
+    {
+        private IWebDriver driver;
```

Comment:
Try to put driver in separate BasePage and extend other pages from it. You won't have to create new driver instance in every page(I hope)